### ISSUE TITLE
iexfinance.utils.exceptions.IEXVersionError: The requested endpoint is only available using IEX Cloud.

### ISSUE THREAD

**Issue by mklueh** [addisonlynch/iexfinance#117]
## Summary (include Python version)
I´m getting an error with the latest version of iexfinance 0.4.0 when using IEX Cloud API endpoints like Stocks, with sandbox as well as default token from IEX CLoud.

### Date/time of issue
2019-03-09

### Expected behavior
According to this reply it seems functions like get_balance_sheet() should be working with version 0.4.0
[(https://github.com/addisonlynch/iexfinance/issues/112#issuecomment-469483110)]

### Actual behavior

Throwing error instead

`iexfinance.utils.exceptions.IEXVersionError: The requested endpoint is only available using IEX Cloud.`

Is the implementation of it missing in 0.4.0?

**Comment by JCCPort** [comment]
I'm getting this too

**Comment by addisonlynch** [comment]
What API version are you using? To use ``get_balance_sheet`` the ``IEX_API_VERSION`` environment variable must  be set to ``iexcloud-beta``. If it is not, ``IEXVersionError`` is raised, as this method is not available with the legacy API

**Comment by JCCPort** [comment]
Yep that was the issue. `os.environ["IEX_API_VERSION"] = "iexcloud-beta"` fixed it for me
